doc/release/upcoming_changes/23528.compatibility.rst

```diff
@@ -0,0 +1,16 @@
+Binaries build against the NumPy C API now with older NumPy versions
+--------------------------------------------------------------------
```

Comment:
Maybe
```suggestion
By default, the exported NumPy C API is now compatible with NumPy 1.17
----------------------------------------------------------------------
```

Reply:
This meandered a bit.  As far as I guessed, `oldest-support-numpy` would pick `1.19` for NumPy 1.25, so that seemed like the right thing now.  But 1.16-1.19 all are the same API version (since we didn't always bump it in the past).

I previously had written 1.17 everywhere though.  The question is what to write here now, 1.19 as what we would pick, or 1.16 because that is what we end up with anyway?

@rgommers 1.19 was correct for `oldest-support-numpy`?  (because 1.19.x is the first version to support Python 3.9)

Reply:
It doesn't seem to matter indeed. Yes, `1.19.3` is what the default is for `oldest-supported-numpy`. I like 1.19, it just eliminated versions to worry about that are irrelevant anyway.